src/fable-splitter/package.json

```diff
@@ -8,8 +8,8 @@
   "bin": {
     "fable-splitter": "./dist/cli.js"
   },
-  "homepage": "https://github.com/fable-compiler/Fable/tree/master/src/typescript/fable-splitter#readme",
-  "repository": "https://github.com/fable-compiler/Fable/tree/master/src/typescript/fable-splitter",
+  "homepage": "https://github.com/fable-compiler/Fable/blob/master/src/fable-splitter/README.md",
```

Comment:
Can we please keep linking to the repo "index":

- https://github.com/fable-compiler/Fable/tree/master/src/fable-splitter#readme

This is what is linked in general because it allows seeing both the README and the repo files.

Reply:
Like this?